### ISSUE TITLE
Make timeout value configurable by user

### ISSUE THREAD

**Issue by MurzNN** [JOJ0/synadm#8]
When making queries very often I got the timeout error:
```
synadm room details '!room_id:ru-matrix.org'
ERROR Timeout: HTTPConnectionPool(host='localhost', port=8008): Read timed out. (read timeout=7)
```
Seems this is because my server is not so nimble as expected, but it want some love too from this script :)

At now timeout is hard-coded in tgose lines:
https://github.com/JOJ0/synadm/blob/45b1220747a81d76118ade33fc33a6623e5b3236/synadm.py#L60
https://github.com/JOJ0/synadm/blob/45b1220747a81d76118ade33fc33a6623e5b3236/synadm.py#L89
https://github.com/JOJ0/synadm/blob/45b1220747a81d76118ade33fc33a6623e5b3236/synadm.py#L116

So will be good to make timeout configurable via `.config/synadm.yaml` file.

**Comment by JOJ0** [comment]
Thanks for reporting. This is easy to implement and will be addressed as soon as possible.

**Comment by JOJ0** [comment]
this was implemented in a8899a6ce

Issue `synadm config` and set timeout to your needs. Please also report back if the feature works for you and then close this issue. Thanks a lot!

**Comment by JOJ0** [comment]
Well I'll close this since you dont seem to find the time to test. Works for me. Thanks for the suggestion.
